st3/sublime_lib/view_utils.py

```diff
@@ -17,6 +17,9 @@ def new_view(window, **kwargs):
 
     :argument encoding: The encoding that the view should use when saving.
 
+    :argument line_endings: The kind of line endings to use:
+        ``'Unix'``, ``'Windows'``, or ``'CR'``.
```

Comment:
I wonder whether using the respective string representation would be clearer. I mean, the underlying API uses these values, but everything within Python just uses `"\r\n"` or `"\n"` and I like that a lot more.